static/scss/snackbar.scss

```diff
@@ -1,27 +1,28 @@
+@use "@material/snackbar";
+
 .mdc-snackbar {
-  z-index: 10;
-  background-color: $message-component-bg-color;
-  left: 50%;
-  border-radius: $std-border-radius;
+  background-color: transparent;
+  z-index: 50;
 
   @include breakpoint(materialmobile) {
     left: 0;
   }
 
-  &.mdc-snackbar--active {
+  &.mdc-snackbar--open {
     bottom: 10px;
     padding-top: 10px;
     padding-bottom: 10px;
   }
 
-  .mdc-snackbar__text {
+  .mdc-snackbar__surface {
+    background-color: $message-component-bg-color;
+    border-radius: $std-border-radius;
     margin: 5px 0;
     padding: 0;
```

Comment:
```suggestion
    padding: 0;
    
    .mdc-snackbar__label {
      width: 100%;
      text-align: center;
    }
```

Reply:
quick suggestion that makes the label look like this:

![Screenshot from 2020-05-05 10-42-55](https://user-images.githubusercontent.com/6207644/81079143-2a6d7100-8ebd-11ea-8f0b-6dae3afc8d82.png)